user-console/src/main/java/org/pentaho/mantle/client/ui/PerspectiveManager.java

```diff
@@ -290,9 +315,8 @@ private void loadResourceBundle( final HasText textWidget, final IPluginPerspect
       } );
 
     } catch ( Exception e ) {
-      String msg = "Error loading message bundle: " + e.getMessage();
-      log.error( msg, e );
-      Window.alert( msg );
+      Window.alert( "Error loading message bundle: " + e.getMessage() );
+      e.printStackTrace();
```

Comment:
Why remove the logger?

Reply:
At first, I made that change but later realized that I must include this in GWT inherits and didn't find a way. So I left it as it were.